cslang/adt.py

```diff
@@ -79,10 +79,9 @@ def define_type(self, container_type, types):
             raise CSlangError(
                 "Illegal type redefinition for type: {}".format(container_type)
             )
-            
```

Comment:
Probably don’t change white space in unrelated files. This can cause the build to be rejected because the auto-formatted doesn’t like it.